src/enclave/rpchandler.h

```diff
@@ -16,9 +16,17 @@ namespace enclave
   public:
     virtual ~RpcHandler() {}
 
+    // Used by enclave to initialise and tick frontends
```

Comment:
comment should go over `tick`?

Reply:
This comment applies to both `set_sig_intervals()`, `set_cmd_forwarder()` and `tick()`.